lib/internal/peripheral_manager.dart

```diff
@@ -0,0 +1,18 @@
+part of flutter_ble_lib;
+
+abstract class PeripheralManager {
```

Comment:
Shouldn't this be `CentralManager` as the application is in `Central` role?

Reply:
This is an internal interface passed to Peripheral and does not pertain to Central/Peripheral distinction. Perhaps the name should be change to clarify this?

Reply:
I've created an issue for this.